test/integration_tests/conftest.py

```diff
@@ -0,0 +1,36 @@
+# coding=utf-8
```

Comment:
We can create a new folder `benchmark_tests`

Reply:
Sure